.github/workflows/iroha2-no-incorrect-image.yml

```diff
@@ -8,12 +8,12 @@ on:
 
 jobs:
   check:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
What is the need? Is it related to Python installation?

Reply:
If you don't specify the version, the install-python action breaks randomly: https://github.com/hyperledger/iroha/issues/4499

Reply:
Okay. What about `ubuntu-24.04` then?

Reply:
I don't remember why I picked 22.04, don't mind switching to 24.04.